Makefile

```
@@ -299,6 +301,11 @@ bats/test: | guard/program/bats
 	cd tests/make && bats -r *.bats
 	@ echo "[$@]: Completed successfully!"
 
+project/validate:
+	@ echo "[$@]: Ensuring the target test folder is not empty"
+	[ "$$(ls -A $(PROJECT_ROOT))" ] && echo "Not empty" || echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1
```

Success is apparent by make continuing on... :) And we can protect the errexit a bit by using a subshell...
```suggestion
	[ "$$(ls -A $(PROJECT_ROOT))" ] || (echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1)
```

---

updated